packages/teleport/src/components/BannerList/Banner.tsx

```diff
@@ -0,0 +1,85 @@
+/*
+Copyright 2022 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import React, { ReactNode } from 'react';
+import styled from 'styled-components';
+
+import { Box, Flex, Text } from 'design';
+import { Cross, Info, Warning } from 'design/Icon';
+
+type Props = {
+  message: string;
+  // The backend defines the severity as an integer value with the current
+  // pre-defined values: LOW: 0; MEDIUM: 5; HIGH: 10
+  severity: number;
+  id: string;
+  onClose: (id: string) => void;
+};
+
+export function Banner({ id, message = '', severity = 0, onClose }: Props) {
+  return (
+    <Box bg={getBackgroundColor(severity)} p={3}>
+      <Flex alignItems="center">
+        {generateIcon(severity)}
+        <Text bold>{message}</Text>
+        <Close
+          onClick={() => {
+            onClose(id);
+          }}
+          role="button"
+        >
+          <Cross />
+        </Close>
+      </Flex>
+    </Box>
+  );
+}
+
+function generateIcon(severity: number): ReactNode {
+  // severity is checked using ranges to allow additional severities to be added
+  // on the backend without requiring an update to the client.
+  if (severity < 5) {
+    return <Info mr={3} fontSize="3" role="icon" />;
+  }
+  if (severity < 10) {
+    return <Info mr={3} fontSize="3" role="icon" />;
+  }
+  if (severity >= 10) {
+    return <Warning mr={3} fontSize="3" role="icon" />;
+  }
+  return null;
+}
+
+function getBackgroundColor(severity: number): string {
+  // severity is checked using ranges to allow additional severities to be added
+  // on the backend without requiring an update to the client.
+  if (severity < 5) {
+    return 'info';
+  }
+  if (severity < 10) {
+    return 'warning';
+  }
+  if (severity >= 10) {
+    return 'danger';
+  }
+  return 'secondary.light';
+}
+
+const Close = styled.div`
```

Comment:
Since this is a clickable element maybe we should use a button tag here to make it focusable

Reply:
i'd also throw in a "on hover, button gets lighter" or some style

Reply:
I've converted it from having `role="button"` to being an actual button and added a hover style to it.

<img width="110" alt="Screen Shot 2022-08-31 at 10 05 10 AM" src="https://user-images.githubusercontent.com/532033/187725831-6df075dd-74c4-4076-bc4c-61f3c2f3c22c.png">

Reply:
I also added a focussed state 

<img width="114" alt="Screen Shot 2022-08-31 at 10 30 55 AM" src="https://user-images.githubusercontent.com/532033/187731024-550c5114-162b-495f-ac2d-5eab31d64f00.png">